health/drivers/tcp/driver.py

```diff
@@ -154,8 +155,15 @@ def main(es, latest_aggregated_ts=None):
 
     for interval in intervals:
         body = get_request(interval)
-        resp = requests.post("%s/_search?search_type=count" % es,
-                             data=json.dumps(body)).json()
+        resp = requests.post("%s/_search" % es,
```

Comment:
I will rephrase. 

What we need is to make sure that only aggregated data is returned, and that there is no original (not aggregated) records in response. 

Please double check this